src/metabase/sync/analyze/classify.clj

```diff
@@ -48,9 +48,10 @@
   [original-model :- FieldOrTableInstance, updated-model :- FieldOrTableInstance]
   (assert (= (type original-model) (type updated-model)))
   (let [[_ values-to-set] (data/diff original-model updated-model)]
-    (log/debug (format "Based on classification, updating these values of %s: %s"
-                       (sync-util/name-for-logging original-model)
-                       values-to-set))
+    (when (seq values-to-set)
+      (log/debug (format "Based on classification, updating these values of %s: %s"
```

Comment:
@sbelak I think this was a mistake, but I see lots of
```
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 50 'dataflow_batch': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 51 'catalog_product_flat_4': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 52 'catalog_product_entity': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 5 'sales_flat_invoice': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 7 'catalog_product_index_eav': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 8 'eav_form_element': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 9 'sales_flat_shipment_track': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 10 'rating_option': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 12 'rating_title': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 13 'eav_entity_int': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 14 'newsletter_template': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 15 'poll_store': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 16 'review_status': null
04-11 15:05:48 DEBUG analyze.classify :: Based on classification, updating these values of Table 18 'sales_shipping_aggregated_order': null
```
When there are no values to update. I think this is what was originally intended, do you agree?